nibabel/cifti2/parse_cifti2.py

```diff
@@ -152,7 +150,7 @@ def StartElementHandler(self, name, attrs):
             # create cifti2 image
             self.header = Cifti2Header()
             self.header.version = attrs['Version']
-            if LooseVersion(self.header.version) < LooseVersion('2'):
+            if self.header.version not in ("2", "2.0"):
```

Comment:
Is that right - that these are the only possible spellings for the version >= 2?   What about versions > 2?  Why not use `Version('2')` here instead?

Reply:
Can restore rough equivalence, just unclear why to expect future versions to be parseable by the same parser.

Reply:
Do we have some reason to think that Cifti versions won't be in standard version format?   Or, put another way, what's the advantage of restricting ourselves to the strings here?

Reply:
If you think it's likely that Cifti will go to some other version convention, we can just stick with the string comparison and run an error?